src/main/java/uk/gov/hmcts/darts/audio/component/impl/OutboundFileProcessorImpl.java

```diff
@@ -206,4 +245,17 @@ private AudioFileInfo reEncode(AudioFileInfo audioFileInfo)
         return audioOperationService.reEncode(StringUtils.EMPTY, audioFileInfo);
     }
 
+    public static List<List<AudioFileInfo>> convertChannelsListToConcatenationsList(List<List<AudioFileInfo>> channelsList) {
```

Comment:
also existing code - i think it is supposed to be removing directory info

Reply:
perhaps worth investigating to see if required or not

Reply:
it's re-encoding from mp2 to mp3 - so definitely required :-)

Reply:
ok, having. a look at it, it seems to be keeping the number of audio files in each channel to me the same as the one in the first channel. So it may throw a ArrayIndexOutOfBoundsException if a file was missing from one of the other channels, but i guess that would be covered in a separate ticket.
@chrisbellingham-hmcts FYI

Reply:
yes. this was discussed and will be covered in another ticket